--- fceux.orig/src/drivers/sdl/sdl.cpp
+++ fceux/src/drivers/sdl/sdl.cpp
@@ -934,7 +934,7 @@
 	if(gtkIsStarted == true && noGui == 0)
 	{
 		GtkWidget* d;
-		d = gtk_message_dialog_new(GTK_WINDOW(MainWindow), GTK_DIALOG_MODAL,        GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, errormsg);
+		d = gtk_message_dialog_new(GTK_WINDOW(MainWindow), GTK_DIALOG_MODAL,        GTK_MESSAGE_ERROR, GTK_BUTTONS_OK, "%s", errormsg);
 		gtk_dialog_run(GTK_DIALOG(d));
 		gtk_widget_destroy(d);
 	}
